source/scss/1977.scss

```diff
@@ -10,7 +10,7 @@
   filter: contrast(1.1) brightness(1.1) saturate(1.3);
 
   &:after{
-    background: rgba(243, 106, 188, .3);
+    background: rgba(243, 106, 188, 0.3);
```

Comment:
Ahh! I intended to remove them if I have any leading 0's. Thank you for the changes! If you take the leading 0's back out, I'd be happy to merge! :+1: